ports/qt5-declarative/vcpkg.json

```
@@ -10,5 +11,26 @@
     },
     "qt5-imageformats",
     "qt5-svg"
-  ]
+  ],
+  "default-features": [
+    "platform-default-features"
```

You can add `d3d12` directly as a default feature.
```suggestion
    "d3d12"
```

---

No, you cannot.

---

Ok, I noticed the explanation in this comment:https://github.com/microsoft/vcpkg/pull/31294#issuecomment-1537223587.